projects/packages/publicize/src/class-publicize-utils.php

```diff
@@ -106,7 +106,18 @@ public static function is_wpcom() {
 	 */
 	public static function assert_is_wpcom( $method ) {
 		if ( ! self::is_wpcom() ) {
-			throw new \Exception( esc_html( "Method $method can only be called on WordPress.com." ) );
+			throw new \Exception( esc_html( "Method $method can only b`e called on WordPress.com." ) );
```

Comment:
This is probably a mistake.

```suggestion
			throw new \Exception( esc_html( "Method $method can only be called on WordPress.com." ) );
```

Reply:
Yes, thanks